pkg/endpoint/restore_test.go

```diff
@@ -90,7 +90,8 @@ func (ds *EndpointSuite) endpointCreator(id uint16, secID identity.NumericIdenti
 
 var (
 	regenerationMetadata = &regeneration.ExternalRegenerationMetadata{
-		Reason: "test",
+		Reason:            "test",
+		RegenerationLevel: regeneration.RegenerateWithoutDatapath, // Needs datapath load??
```

Comment:
Is this variable used at all?

Reply:
Maybe not, but I'd like to keep `RegenerationLevel` explicitly initialized also in tests, in case someone ever copy&pastes it somewhere else.

Reply:
Removed the comment.